res/skins/QMLDemo/Mixxx/Controls/Knob.qml

```diff
@@ -63,30 +64,51 @@ Item {
     DragHandler {
         id: dragHandler
 
-        property real valueRange: root.max - root.min
+        property real value
         property vector2d lastTranslation: Qt.vector2d(0, 0)
 
         target: null
         grabPermissions: PointerHandler.CanTakeOverFromAnything | PointerHandler.ApprovesTakeOverByAnything
-        onActiveChanged: lastTranslation = Qt.vector2d(0, 0)
+        onActiveChanged: {
+            dragHandler.value = root.value;
+            lastTranslation = Qt.vector2d(0, 0);
+        }
         onTranslationChanged: {
             const delta = lastTranslation.y - translation.y;
-            const change = valueRange * Math.max(Math.min(delta, 100), -100) / 100;
-            const value = Math.max(root.min, Math.min(root.max, root.value + change));
+            const change = (root.max - root.min) * Mixxx.MathUtils.clamp(delta, -100, 100) / 100;
+            const value = Mixxx.MathUtils.clamp(root.value + change, root.min, root.max);
             lastTranslation = translation;
             root.turned(value);
-            root.value = value;
+            dragHandler.value = value;
         }
     }
 
+    Binding {
+        when: dragHandler.active
+        target: root
+        property: "value"
+        value: dragHandler.value
+    }
+
     MouseArea {
+        id: wheelHandler
+
+        property real value
+
         anchors.fill: parent
         acceptedButtons: Qt.NoButton
         onWheel: {
             const value = (wheel.angleDelta.y < 0) ? Math.min(root.max, root.value + 0.1) : Math.max(root.min, root.value - 0.1);
```

Comment:
~Whats the reason for the 0.1 offsets?~
Can you put create a named variable (eg `stepSize`) for it?